bundle-workflow/src/read_write_s3.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+
+import boto3
+
+
+class ReadWriteFiles:
+    """
+    get_S3Objects:Used for getting the files from S3
+    bucket_name: Name of the bucket you want to get the file from
+    role_arn: Amazon Resource Name (ARN) of the role that you want to assume
+    Here, default value for role is taken. Can be changed through -a <role-name>
+    role_session_name: Name for the session
+    Here, default value for session is taken. Can be changed through -s <session-name>
+    bucket_bundle: Object path from where you want to get the bundle
+    path_to_save_bundle: Path where you want to save the bundle at
+    """
+    def __init__(self, role_arn, role_session_name):
+        self.role_arn = role_arn
+        self.role_session_name = role_session_name
+
+    def get_S3_objects(self, bucket_name, bucket_bundle, path_to_save_bundle=None):
```

Comment:
Rename to `get` or `download`.

Reply:
The parameters `bucket_bundle` and `path_to_save_bundle` don't make sense in S3 context. Should be `prefix` and `dest`. 

What's a `None` dest? I would just require it.